phpBB/includes/acp/acp_styles.php

```diff
@@ -1123,7 +1123,13 @@ public function sort_styles($style1, $style2)
 	*/
 	protected function read_style_cfg($dir)
 	{
+		if (!file_exists($this->styles_path . $dir . '/style.cfg'))
+		{
+			trigger_error($this->user->lang['NO_STYLE_CFG'] . $dir, E_USER_WARNING);
```

Comment:
preferred way would be:
`$this->user->lang('NO_STYLE_CFG', $dir)`
and 
`'Could not get the style configuration file for: %s'`

Reply:
Yup, thx. Let me test it first.